db/migrate/20190607014201_add_column_approved_to_institutions.rb

```diff
@@ -1,5 +0,0 @@
-class AddColumnApprovedToInstitutions < ActiveRecord::Migration
```

Comment:
@jholton - I'm sure you already considered this, but want to confirm.  Prior to this PR being merged, you will have ensured that the two deleted migrations are migrated `down` in all envs (i.e. dev/staging/prod).

Reply:
Yup, done already.